scripts/ci/testing/run_offline_sql_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# shellcheck source=scripts/ci/libraries/_script_init.sh
+. "$( dirname "${BASH_SOURCE[0]}" )/../libraries/_script_init.sh"
+
+
+build_images::prepare_ci_build
+
+build_images::rebuild_ci_image_if_needed_with_group
+testing::get_docker_compose_local
+testing::setup_docker_compose_backend "offline-sql-test"
+# We test from 2.2.0 because we can't run offline test for MSSQL for airflow versions below 2.2.0
+testing::run_command_in_docker "offline-sql-test" "airflow db upgrade -r 2.2.0:2.2.4 \
```

Comment:
I will update this command once https://github.com/apache/airflow/pull/22102 is merged